addons/snailmail_account/wizard/account_move_send.py

```diff
@@ -13,31 +13,38 @@ class AccountMoveSend(models.Model):
         store=True,
         readonly=False,
     )
-    send_by_post_warning_message = fields.Text(compute='_compute_send_by_post_warning_message')
+    send_by_post_cost = fields.Integer(string='Stamps', compute='_compute_send_by_post_extra_fields', readonly=True)
```

Comment:
```suggestion
    send_by_post_cost = fields.Integer(string='Stamps', compute='_compute_send_by_post_extra_fields')
```
a computed field is already readonly by default